src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -272,6 +272,13 @@ class ProcurationRequest
     /**
      * @var bool
      *
+     * @Assert\IsTrue(message="procuration.authorization.required", groups={"front"})
+     */
+    public $authorization;
```

Comment:
We do not need such public property on the entity, we can use the `mapped` option set to `false` on the form fields instead.